doc/staging/api-boilerplate-templates.md

```diff
@@ -14,6 +14,8 @@ API boilerplate templates v1.0.0
 
 ### Features
 
+![node-mongo](https://github.com/KamiNation/node-mongo/blob/develop/doc/staging/node-mongo-cli.gif?raw=true)
+
```

Comment:
Update on your PR @KamiNation 

#### Notification related
Please tag me in your pull request, otherwise I might not receive a notification that you sent a PR. I didn't receive a notification I had to look for your PR .

#### Community/contribution related
The community doc https://code-collabo.gitbook.io/community-doc-v1.0.0/collabo-guidelines/contributing specified how you should name your PRs, and also that you should link the issue you fixed to your PR. You did not do this. It makes it hard for me to refer back to the issue when I need to.
Do this for this PR and every other PR you submit in future.

#### Task related
There are a number of things I want to point out about the gif. But first, nice job on how you were able to follow those steps without issues in the video. That said, here are some thoughts:
- Color contrast: I can barely see the gif content well due to the blue gradient wallpaper, with the not so dark terminal and the not so sharp colors in the terminal etc.
- Pace: The pace at which everything went is very slow including typing, and I think it's because you were trying to be careful. 

Anyway, the slow pace is not much of a problem, because one can fast forward a bit with some software/app.

My main problem is the colour contrast I mentioned earlier. Because of this, I might have to do this gif video myself. Which is why I wrote on the channel that you could pick up another issue to work on. I would take over this one.

Reply:
Okay. I can work on that.

Reply:
All right. I'll leave you to finish it then. Thank you 🙂